subsys/bluetooth/host/buf.c

```diff
@@ -78,7 +74,7 @@ struct net_buf *bt_buf_get_rx(enum bt_buf_type type, k_timeout_t timeout)
 		buf = net_buf_alloc(&acl_in_pool, timeout);
 	}
 #else
-	buf = net_buf_alloc(&hci_rx_pool, timeout);
+	buf = net_buf_alloc(&evt_pool, timeout);
```

Comment:
I don't completely understand this change. First of all, it's a bit unintuitive that you'd be allocating ACL buffers from a pool called `evt_pool`. Second of all, we have separate Kconfig options for ACL and event buffer counts and sizes, but it seems we're generalising those into a single pool? (it seems this may have been the case before this PR as well, though). This might result in unexpected behaviour, since the user (who creates an application configuration) might think that there's always `CONFIG_BT_BUF_ACL_RX_COUNT` buffers available for ACL packets, but that will not be the case if there are lots of events coming in and depleting available buffers in the pool.

Reply:
You are right, I misunderstood something